.github/workflows/backend.yml

```
@@ -81,13 +81,15 @@ jobs:
     name: Build
     runs-on: ubuntu-latest
     timeout-minutes: 30
-    needs: [ build ]
+    needs: [ build, paths-filter ]
     if: always()
     steps:
       - name: Status
         run: |
-          if [[ ${{ needs.build.result }} == 'success' || ${{ needs.paths-filter.outputs.ignore == 'true' }} ]]; then
+          if [[ ${{ needs.build.result }} == 'success' ]]; then
             echo "Passed!"
+          elif [[ ${{ needs.paths-filter.outputs.not-ignore }} == 'false' ]]; then
+            echo "Skip Build!"
           else
             exit -1
           fi
```

Do you think this 👇 would be easier to understand?

```suggestion
          if [[ ${{ needs.paths-filter.outputs.ignore }} == 'true' ]]; then
            echo "Skip Build!"
            exit 0
          fi
          if [[ ${{ needs.build.result }} != 'success' ]]; then
            echo "Failed to build"
            exit -1
          fi
          # If we need to add more condition in the future, we can simply add
          if [[ ${{ needs.new.condition.result }} != 'success' ]]; then
            echo "Failed to do some new stuff"
            exit -1
          fi
```

---

It's better. I'll fix it.